osdep/mac/app_bridge.h

```diff
@@ -29,9 +29,10 @@ enum {
 };
 
 enum {
-    RENDER_TIMER_CALLBACK = 0,
-    RENDER_TIMER_PRECISE,
+    RENDER_TIMER_PRESENTATION_FEEDBACK = -1,
     RENDER_TIMER_SYSTEM,
+    RENDER_TIMER_CALLBACK,
```

Comment:
Why swap system and callback?

Reply:
it also shouldn't be an issue to change the numeric values, since i use the enum directly and not their numeric representation. this change is also not user facing.

the reasoning for the reordering itself was the grouping of similar working mechanism:
RENDER_TIMER_CALLBACK + RENDER_TIMER_PRECISE use a locking/waiting/signal mechanism on swap for vsynching. RENDER_TIMER_SYSTEM doesn't do any manual synching and relies on the Metal layers sync mechanism + nextDrawable object. RENDER_TIMER_PRESENTATION_FEEDBACK is just RENDER_TIMER_SYSTEM + the feedback. admittedly they would have been grouped too, if i had just added it to the end of the enum. though i would prefer to keep the none locking ones as <=0 and the locking ones as >0.

like this i can easily check if i need the locking/waiting/signal with `if option.mac.macos_render_timer > RENDER_TIMER_SYSTEM`.

Reply:
Ok, as long as there is no weird way to access numeric values of enum options it is ok.